ambry-frontend/src/main/java/com/github/ambry/frontend/FrontendUtils.java

```diff
@@ -37,11 +40,17 @@
 import org.json.JSONTokener;
 import org.slf4j.Logger;
 
+import static com.github.ambry.rest.RestUtils.InternalKeys.*;
+
 
 /**
  * Common utility functions that will be used across frontend package
  */
 class FrontendUtils {
+  /**
+   * Path separator character for named blob uri.
+   */
+  private static final char PATH_SEPARATOR_CHAR = '/';
```

Comment:
can be replace by RestUtils.Path_SEPARATOR_CHAR.

Reply:
updated.